packages/amplify_datastore_plugin_interface/lib/src/types/models/model_index.dart

```diff
@@ -0,0 +1,77 @@
+/*

```

Comment:
based on talks with William + Daniel we will not be supporting indexes for Datastore in this version as we have found that Android and iOS have significant differences in implementation.

Reply:
Okies